plugins/generator-1.19.4/forge-1.19.4/cmdargs/_call_procedure.java.ftl

```diff
@@ -0,0 +1,17 @@
+<#include "procedures.java.ftl">
+.executes(arguments -> {
+    ServerLevel world = arguments.getSource().getLevel();
+
+    double x = arguments.getSource().getPosition().x();
+    double y = arguments.getSource().getPosition().y();
+    double z = arguments.getSource().getPosition().z();
+
+    Entity entity = arguments.getSource().getEntity();
+    if (entity == null)
+        entity = FakePlayerFactory.getMinecraft(world);
+
+	Direction direction = entity.getDirection();

```

Comment:
This indentation is probably one level too deep, check other files too

Reply:
Because this line starts with <kbd>\\t</kbd>, which is seen as 4 spaces in IDEA and as 8 (by default) spaces on GitHub.

Reply:
> This indentation is probably one level too deep, check other files too

I don't know. It is the same files as 1.19.2.